### PR TITLE
Enable certmonger to restart on reboot

### PR DESCRIPTION
**Pull request by jvlcek**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1670604

### CODE REVIEW

**Comment by jvlcek** [comment]
This PR ensures the certmonger service is _enabled_ allowing it to restart automatically on reboot.

One way to test this is to use the appliance_console to request a new ipa cert with the following
command:
`
appliance_console_cli   --ca=ipa --http-cert
systemctl status certmonger | grep "certmonger.service;"
`

Ensure the output show enabled. e.g.:
`Loaded: loaded (/usr/lib/systemd/system/certmonger.service; **_enabled_**; vendor preset: disabled)`

Reboot te appliance and comfirm that the certmonger service restarted on reboot by 
examining the output of:

`systemctl status certmonger | grep "certmonger.service;"`

**Comment by jvlcek** [comment]
@miq-bot add_label bug

**Comment by jvlcek** [comment]
@miq-bot assign @carbonin

**Comment by jvlcek** [comment]
@carbonin  Please review.

**Comment by jvlcek** [comment]
@kbrock Please review

**Comment by jvlcek** [comment]
Thank you @kbrock 
@carbonin and @gtanzillo Please take a look and merge if appropriate.

File: lib/manageiq/appliance_console/certificate_authority.rb [hunk 1]
```diff
@@ -92,6 +98,9 @@ def configure_pgserver
         # no need for username/password since not writing database.yml
         InternalDatabaseConfiguration.new(:ssl => true).configure_postgres
         LinuxAdmin::Service.new(PostgresAdmin.service_name).restart
+
+        say "configuring certmonger to start on reboot"
+        LinuxAdmin::Service.new("certmonger").enable.start
```
Comment (carbonin) [inline lib/manageiq/appliance_console/certificate_authority.rb]: Should this maybe be a class method on the `Certificate` class?

Either way, I think making it a method somewhere would be better than this same call being in three places.

File: spec/certificate_authority_spec.rb [hunk 2]
```diff
@@ -28,6 +28,58 @@
...
+      subject.pgclient = true
+    end
+
+    it "without ipa client should not install" do
+      ipa_configured(false)
+      expect { subject.activate }.to raise_error(ArgumentError, /ipa client/)
+    end
+
+    it "should configure postgres client" do
+      ipa_configured(true)
```
Comment (carbonin) [inline spec/certificate_authority_spec.rb]: Didn't we have a conversation that ended with us deciding that IPA had nothing to do with PG certs? If that's the case, I don't understand this logic.

**Comment by jvlcek** [comment]
@carbonin Thank you again for the input.  I think this addresses the issues you raised.

**Comment by jvlcek** [comment]
@carbonin, @kbrock & @gtanzillo I believe I've address all the raised issues. Can this be merged now?

Thank you, JoeV

**Pull request merged**